src/content/app/new-species-selector/components/species-search-results-summary/SpeciesSearchResultsSummary.scss

```diff
@@ -9,3 +9,21 @@
 .searchMatchesCount {
   font-weight: $bold;
 }
+
+.noMatchesMessage {
+  color: $red;
+  margin-top: 40px;
+  margin-bottom: 30px;
+}
+
+.searchHelp p {
+  margin: 0;
+}
+
+.searchHelp ul {
+  padding-inline-start: 20px;
+}
+
+.searchHelp li {
```

Comment:
This overrides the `list-style-type` in `ul` (Normalize.css) so how about moving it to `searchHelp ul`?

Reply:
I don't think it matters. `list-style-type` property applies to `li` element anyway. Setting it on `ul` in normalize was using the css cascade to pass that rule down to the `li`s. Setting it here directly on `li` feels more natural.

See MDN:

> Only a few elements (<li>) and <summary>) have a default value of display: list-item. However, the list-style-type property may be applied to any element whose display value is set to list-item. Moreover, because this property is inherited, it can be set on a parent element (commonly <ol> or <ul>) to make it apply to all list items.